src/regex.js

```diff
@@ -39,6 +39,9 @@ SVG.regex = {
   // Test for image url
 , isImage:          /\.(jpg|jpeg|png|gif|svg)(\?[^=]+.*)?/i
 
+  // Test for url reference
+, isUrl:            /url\(([-\w:/]+(?:\.\w+)?)?#([-\w]+)\)/
```

Comment:
I think there might be some cases that this regexp is not matching, like `url(https://example.org/absolute/URI/with/absolute/path/to/resource.txt#theid)`. Also, I was wondering if we could use a simpler regexp instead, something like this:  `/url\((.*)#([-\w]+)\)/`. We keep it simple and just check if there is something in front of the id. If there is nothing, then we had the complete url. I don't know if it is worth it to try to detect if the path before the id is absolute or relative.